### ISSUE TITLE
Incorrect function signature

### ISSUE THREAD

**Issue by bpas247** [UWB-ACM/CTCI#6]
in string manipulation file for the function signature for problem 2,

```c++
#import <string>
using namespace std;

string englishInteger(int num);
```
should be 
```c++
#include <string>
using namespace std;

string englishInteger(int num);
```

**Comment by etcadinfinitum** [comment]
lol OOPS, my bad

**Comment by etcadinfinitum** [comment]
will fix this and the issue with #1 ... sometime soon

**Comment by etcadinfinitum** [comment]
Closing this; fixed in #7
